templates/header.war.html

```diff
@@ -11,12 +11,19 @@
 <p>
   You will need to pre-install a supported version of Java:
 <p>
-
-<ul>
-  <li>2.164 (2019-02) and newer: Java 8 or Java 11</li>
-  <li>2.54 (2017-04) and newer: Java 8</li>
-  <li>1.612 (2015-05) and newer: Java 7</li>
-</ul>
+          <b> <a href="https://www.jenkins.io/download/lts/">Long Term Support (LTS)</a> Version</b>
+          <ul>
+                <li>2.346 (2022-06) and newer: Java 8, Java 11, or Java 17</li>
+                <li>2.164 (2019-02) and newer: Java 8 or Java 11</li>
+                <li>2.54 (2017-04) and newer: Java 8</li>
+                <li>1.612 (2015-05) and newer: Java 7</li>
```

Comment:
This list makes no sense. Except for 2.346, none of the versions listed are LTS baselines. None of the versions listed are LTS versions.

If you're going to split this into two lists, I recommend all of this content be moved to the weekly list, and a new list with actual LTS versions be added.